### DISCUSSION TITLE
[Question]: What mods would I disable to only have Possession Mod?

### DISCUSSION THREAD

**Discussion by 1-Mythical** [winstxnhdw/lc-hax#56] [Q&A] [answered]
What mods should be disabled to still have Possession mod and not everything else?

**Comment by winstxnhdw** [comment, marked as answer]
Phantom, Trigger and Possession.

**Reply by 1-Mythical** [reply]
Those mods should be disabled?

**Reply by winstxnhdw** [reply]
Sorry, I misread. These are the only mods you need.
